docs/samples/browser-plugin-meetings/app.js

```diff
@@ -3330,6 +3330,26 @@ function toggleBreakout() {
   }
 }
 
+function beRightBackTrue() {
+  const meeting = getCurrentMeeting();
+
+  meeting.beRightBack(true).then((res) => {
+    console.log(`meeting.beRightBack(true) done successfully with result: ${res}`);
+  }).catch((err) => {
+    console.log(`meeting.beRightBack(true) done unsuccessfull with err: ${err}`);
+  });
+}
```

Comment:
Why not just

```suggestion
function beRightBack(enabled) {
  const meeting = getCurrentMeeting();

  meeting.beRightBack(enabled).then((res) => {
    console.log(`meeting.beRightBack(${enabled}) done successfully with result: ${res}`);
  }).catch((err) => {
    console.log(`meeting.beRightBack(true) done unsuccessfull with err: ${err}`);
  });
}
```

And later
```typescript
<button id="beRightBack" type="button" onclick="beRightBack(true)">Step Away true</button>
```

Reply:
I completed this part in the simplest way possible and created a task to update it properly in https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-574653

Reply:
Could we throw in a TODO or something in the code with a link to the JIRA so we have a reference to it?

Reply:
@brycetham  @SomeBody16  Added Todo